lib/restaurant.rb

```diff
@@ -0,0 +1,25 @@
+require_relative 'menu'
+require_relative 'send_sms.rb'
+
+class Restaurant
+  def initialize(menu = Menu.new)
+    @menu = menu
+  end
+
+  def check_menu
+    @menu.list_food
+  end
+
+  def place_order
+    @menu.make_order
```

Comment:
great again to use a function to place the order which is calling the method from the menu class